engine/table/src/main/java/io/deephaven/engine/table/impl/select/FilterSelectColumn.java

```diff
@@ -107,6 +113,18 @@ private List<String> checkForInvalidFilters() {
                     "Cannot use a filter with virtual row variables (i, ii, or k) in select, view, update, or updateView: "
                             + filter);
         }
+        if (filter.isRefreshing()) {
```

Comment:
Maybe this means that when we gather filters that are dependencies in QueryTable or WouldMatch, we should only consider refreshing filters.